### PR TITLE
20200120-drivetrain merge with master

### PR DESCRIPTION
**Pull request by LeviWalker**
Merge 20200120-drivetrain into 20200130-drivetrain

### CODE REVIEW

File: src/main/java/xyz/nasaknights/infiniterecharge/commands/drivetrain/VisionDriveAssistCommand.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+/**
+ * This command will be used for controlling the drivetrain with the data from the Vision Client.
+ *
+ * @see xyz.nasaknights.infiniterecharge.subsystems.DrivetrainSubsystem
+ */
+public class VisionDriveAssistCommand extends CommandBase
+{
+    public VisionDriveAssistCommand()
+    {
```
Comment (bhooten) [inline src/main/java/xyz/nasaknights/infiniterecharge/commands/drivetrain/VisionDriveAssistCommand.java:13]:
```suggestion
    {
        addRequirements(RobotContainer.getDrivetrain());
```

File: src/main/java/xyz/nasaknights/infiniterecharge/commands/drivetrain/VisionDriveAssistCommand.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package xyz.nasaknights.infiniterecharge.commands.drivetrain;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
```
Comment (bhooten) [inline src/main/java/xyz/nasaknights/infiniterecharge/commands/drivetrain/VisionDriveAssistCommand.java]:
```suggestion
import xyz.nasaknights.infiniterecharge.RobotContainer;
```

**Review by bhooten — APPROVED** [review]

**Review by bhooten — CHANGES_REQUESTED** [review]
Changes need to be tested before committing

**Comment by LeviWalker** [comment]
This code has been tested on the robot and is ready to merge.

**Comment by bhooten** [comment]
This branch is stale and was superseded in PR #18. Closed.

**Pull request merged**